internal/app/clients.go

```diff
@@ -64,3 +65,25 @@ func NewConditionsClient(ctx context.Context, cfg *model.ConfigOIDC, reauth bool
 		co.WithAuthToken(token),
 	)
 }
+
+func NewBomServiceClient(ctx context.Context, cfg *model.ConfigOIDC, reauth bool) (*bomclient.Client, error) {
+	if cfg == nil {
+		return nil, errors.Wrap(ErrNilConfig, "missing bom service API client configuration")
+	}
+
+	if cfg.Disable {
+		return bomclient.NewClient(
+			cfg.Endpoint,
+		)
+	}
+
+	token, err := auth.AccessToken(ctx, model.BomsServiceAPI, cfg, reauth)
```

Comment:
This code looks fine, but there is some setup we'll have to do to get BOMService to get things to authenticate properly, unless we're using the same Okta groups and stuff for BOMService as we do for ConditionOrc (and that would probably be fine).

Reply:
Thanks for pointing it out.

I am not familiar with okta groups. Currently I found oidc client id `0oaa8kto5lZYascqP5d7`, does it matches a group in okta? Can you help provide me a link to the okta configuration that I can learn and play around? In the future we may need to have a separate group for bom service.

Thanks!